New_website_code/website/templates/banner.html

```diff
@@ -1,9 +1,10 @@
 {% load static %}
 <div id="banner_container">
     <div class="mt-[9rem]" id="banner_con">
-        <div class="md:hidden block text-center text-white w-4/5 m-auto">
-            <h2 class="text-[#80AB9E] text-xl">A minimal schema </h2>
-            <p class="text-3xl font-bold">for expressing linked trust claims permissionlessly</p>
+                    <!-- moka work here to update the text  -->
+        <div class="text-center text-white w-4/5 m-auto">
+            <h2 class="text-[#80AB9E] text-xl">LinkedTrust</h2>
+            <p class="text-3xl font-bold">Expressing trust claims permissionlessly</p>
         </div>
```

Comment:
Hey moka actually i think we just want to delete this banner and wherever it is used

its not part of the current design